src/audio/module_adapter/module/iadk_modules.c

```diff
@@ -249,6 +312,13 @@ static int iadk_modules_get_configuration(struct processing_module *mod, uint32_
 					  uint32_t *data_offset_size, uint8_t *fragment,
 					  size_t fragment_size)
 {
+	if (mod->is_native_sof) {
+		struct module_interface *mod_in =
+					(struct module_interface *)mod->priv.module_adapter;
+
+		return mod_in->get_configuration(mod, config_id, data_offset_size,
+						 fragment, fragment_size);
+	}
```

Comment:
ditto, why is this call different for other IADK calls ? in that we call both native and IADK APIs ? 
Best to explain in inline comments.